### DISCUSSION TITLE
Do you need a sound card if you are using RTSP?

### DISCUSSION THREAD

**Discussion by killervette42** [tphakala/birdnet-go#168] [General]
Do you need a sound card if you are using RTSP? I am running a VM without a sound card and docker won't start because it can't find the default system sound card, sys default.

**Comment by isZumpo** [comment]
Hi, I'm quite sure there should not be any need for a sound card if you are listening to a RTSP stream. Do you mind sharing what commands you are currently using to start birdnet-go, in addition it would be very helpful with the error message

**Reply by PovilasID** [reply]
I ran in docker so I let it create `config.yaml` file and in it set
```
  audio:
    source: "rtsp" 
```
I think by default it is 'systemdefault' or something.

**Comment by tphakala** [comment]
> Do you need a sound card if you are using RTSP? I am running a VM without a sound card and docker won't start because it can't find the default system sound card, sys default.

No you do not need soundcard for RTSP only, just leave --device /dev/snd and --env ALSA_CARD parameters out when starting container for RTSP

**Comment by killervette42** [comment]
Ok thanks, will take a look again.
